docs/stressng.md

```diff
@@ -94,7 +94,7 @@ Note: this is currently in preview mode.
 ```
 
 The above is the additional changes required to run stressng in vms.
-Currently, we only support images that can be used as [containerDisk](https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#containerdisk).
+Currently, we only support images that can be used as [containerDisk](https://docs.openshift.com/container-platform/4.6/virt/virtual_machines/virtual_disks/virt-using-container-disks-with-vms.html#virt-preparing-container-disk-for-vms_virt-using-container-disks-with-vms).
```

Comment:
@ebattat Can we split these into two PRs? This one for hammerdb, another one for stressng?

Reply:
We can but there are only few changes for stressng (left over from last from merge) .
Regarding the link above it didnt work, so I put a new correct one.